src/context/Data/data-context.js

```diff
@@ -0,0 +1,30 @@
+import { createContext, useContext, useReducer } from 'react';
+import { DataReducer, initialState } from '../../reducer/DataReducer';
+import { getNotes } from '../../services/Services';
+import { useAuth } from '../auth/auth-context';
```

Comment:
If you have time, you can read about barrel exports and absoloute paths, it will make the imports shorter. do it after you complete basic requirements

Reply:
Sure, I'll make changes